### ISSUE TITLE
Plans for Umbraco 14?

### ISSUE THREAD

**Issue by leddt** [ArnoldV/Our.Umbraco.GMaps#173]
Hi, we've been using this package in a few Umbraco 13 projects. We are looking into upgrading them to Umbraco 14.

Umbraco 14 introduced a major breaking change by dropping Angular JS from the backoffice. From what I understand, this breaks most backoffice plugins such as this one.

Do you have plans to port this package to the new web-components-based system?

**Comment by sandromastronardi** [comment]
I'm wondering the same. picked up an old project and wanted to upgrade, now run in to problems because it references old backoffice, but while reading this i realize it will probably completely not work with the new system.

**Comment by robertjf** [comment]
Yes, there is a forked project that is targetting the new uUI but hasn't been completed - see here for a bit more detail:

https://github.com/nul800sebastiaan/UmbraCollabBellissimaGmaps/issues/1#issuecomment-2548005430

We do plan on getting this resolved but I don't have a lot of free time at the moment

**Comment by arknu** [comment]
@robertjf We are going to do a v15/v16 port (since v14 is largely irrelevant now) of this package, since we have a number of sites using it. We are just starting it now, but we'll be contributing it as a PR to this project once it is ready. It should hopefully be ready later this month. 

The fork you mentioned appears to be quite old, so I'll probably not use it directly, but will of course use it as inspiration.

I hope you'll be able to do merge the PR once it is ready and make a release so that other users can benefit.

**Comment by arknu** [comment]
@robertjf I have created a draft PR here. It is not quite done yet, but the basics are working: https://github.com/ArnoldV/Our.Umbraco.GMaps/pull/190

**Comment by arknu** [comment]
@robertjf PR is now ready for review, everything should be implemented for v15 now.

I'm going to use this version in a couple of sites I'm upgrading and will of course push any needed fixes to the PR as needed.

**Comment by arknu** [comment]
@robertjf When are you releasing the final of v4? I'll probably start working on the v16 version soon, hope to have it ready in July. There are a few breaking changes in v16, so we might have to do a v5 for that. So it would be nice to have a final v4 out first...
